omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/AlreadyPaged.java

```diff
@@ -31,12 +31,19 @@
 	
 	private boolean hasMoreResults;
 	
+	private int totalCount;

```

Comment:
It's better to make it Long to be prepared for huge tables and make it an object instead of a primitive so it is null and not 0 when not defined.